.chloggen/fix_prometheusexporter-shutdown-server.yml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
it looks like the presubmit expects this file to be named with a `.yaml` suffix instead of a `.yml` suffix.  Please update